microsoft/testsuites/dpdk/dpdk.yml

```diff
@@ -0,0 +1,5 @@
+name: dpdk
```

Comment:
Don't commit the runbook for testing. You can put it in some ignored folder to avoid it's included.

Reply:
Is there a reason we don't include these? It was sort of a pain to figure out and for others developing tests it might help to have examples that select single tests or run specific tests on other platforms.

Reply:
We want to provide simple experience to users, who run test cases or who develop test cases. It's a good feedback that developer need know how to debug one case.
Would you file an issue to track it? We can add document to help on it.
The current dpdk.yml applies to dpdk+azure only, it's not extendable. We need a yaml file for one case and other platforms.

Reply:
This file can be put in .vscode or other ignored folders, so that won't be submitted incidentally.

Reply:
will remove